### ISSUE TITLE
Distribution of work clustes of versions

### ISSUE THREAD

**Issue by BastienDZ** [NovaFrost/SHS100K#3]
Currently the work clusters of versions are separated per subset train / validate / test.
Is there any specific reason for this? Why not distribute the versions over the 3 sets regardless of their work?

**Comment by NovaFrost** [comment]
> Currently the work clusters of versions are separated per subset train / validate / test. Is there any specific reason for this? Why not distribute the versions over the 3 sets regardless of their work?

I consider cover song identification to be a retrieval task. Therefore, I consolidate all versions into a single set to evaluate the model's performance on a test set that does not contain any versions from the training or validation sets.
